torch/onnx/symbolic_opset9.py

```diff
@@ -723,6 +724,29 @@ def atan(g: jit_utils.GraphContext, self):
     return g.op("Atan", self)
 
 
+@_onnx_symbolic("aten::atan2")
+@_beartype.beartype
+def atan2(g: jit_utils.GraphContext, self, other):
+    # self is y, and other is x on coordinate
+    slope = g.op("Div", self, other)
```

Comment:
what about 0s in other?

Reply:
That should be included as arctangent(inf) is valid. I guess the case is included in OP_DB, but not sure.

Reply:
Both torch/numpy gives inf, so if ONNX gives Nan for example, I think that should be corrected.

Reply:
> However, is 0 in denominator beyond ONNX spec?

Not sure, I suppose it should be inline with torch.